<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessClass</class>
 <widget class="QMainWindow" name="ChessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>中国象棋</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Chess.qrc">
    <normaloff>:/Chess/Resources/icon.png</normaloff>:/Chess/Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="centralLayout">
    <item>
     <widget class="QFrame" name="gameFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="gameLayout">
       <item>
        <widget class="QLabel" name="chessBoard">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>540</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>540</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(139, 69, 19);
border: 2px solid black;</string>
         </property>
         <property name="text">
          <string>棋盘</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="playerInfoLayout">
         <item>
          <widget class="QFrame" name="player1Frame">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(100, 149, 237);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="player1Layout">
            <item>
             <widget class="QLabel" name="player1Label">
              <property name="styleSheet">
               <string notr="true">color: white; font-weight: bold;</string>
              </property>
              <property name="text">
               <string>红方</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="player1Time">
              <property name="styleSheet">
               <string notr="true">color: white; font-size: 18px; font-weight: bold;</string>
              </property>
              <property name="text">
               <string>00:00:00</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="player2Frame">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(139, 69, 19);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="player2Layout">
            <item>
             <widget class="QLabel" name="player2Label">
              <property name="styleSheet">
               <string notr="true">color: white; font-weight: bold;</string>
              </property>
              <property name="text">
               <string>黑方</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="player2Time">
              <property name="styleSheet">
               <string notr="true">color: white; font-size: 18px; font-weight: bold;</string>
              </property>
              <property name="text">
               <string>00:00:00</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionFlipBoard"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>新游戏</string>
   </property>
  </action>
  <action name="actionFlipBoard">
   <property name="text">
    <string>翻转棋盘</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Chess.qrc"/>
 </resources>
 <connections/>
</ui>
